package.json

```diff
@@ -19,18 +19,20 @@
   },
   "homepage": "https://github.com/broccolijs/broccoli",
   "dependencies": {
-    "broccoli-kitchen-sink-helpers": "^0.2.0",
-    "broccoli-slow-trees": "^1.1.0",
-    "commander": "^2.0.0",
-    "connect": "^3.2.0",
+    "broccoli-kitchen-sink-helpers": "^0.2.5",
+    "broccoli-slow-trees": "^1.0.0",
+    "commander": "^2.5.0",
+    "connect": "^3.3.3",
     "copy-dereference": "^1.0.0",
-    "findup-sync": "^0.1.2",
+    "findup-sync": "^0.1.3",
     "handlebars": "^2.0.0",
+    "jshint": "^2.5.10",
     "mime": "^1.2.11",
-    "promise-map-series": "^0.2.1",
     "quick-temp": "^0.1.2",
     "rimraf": "^2.2.8",
-    "rsvp": "^3.0.6",
+    "promise-map-series": "^0.2.1",
+    "rsvp": "^3.0.14",
+    "tap": "^0.4.13",
```

Comment:
Hm, some packages seem to have been moved from devDependencies to dependencies. Are these upgrades actually necessary?